amps/ags/rm-community/rm-community-repo/pom.xml

```diff
@@ -548,7 +548,7 @@
                    <configuration>
                       <images>
                          <image>
-                            <name>${image.name}:${image.tag}</name>
+                             <name>${image.name}:${image.tag}</name>
```

Comment:
Whitespace change made tags not aligned.

Reply:
Done